packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -595,3 +613,8 @@ exports.ApplicationCommand = ApplicationCommand;
  * @external ApplicationCommandOptionAllowedChannelTypes
  * @see {@link https://discord.js.org/docs/packages/builders/stable/ApplicationCommandOptionAllowedChannelTypes:TypeAlias}
  */
+
```

Comment:
This goes in the [APITypes](https://github.com/discordjs/discord.js/blob/main/packages/discord.js/src/util/APITypes.js) file